polymorphia-backend/src/main/java/com/agh/polymorphia_backend/service/mapper/ItemMapper.java

```diff
@@ -0,0 +1,90 @@
+package com.agh.polymorphia_backend.service.mapper;
+
+import com.agh.polymorphia_backend.dto.request.course.reward.item.FlatBonusRequestItemRequestDto;
+import com.agh.polymorphia_backend.dto.request.course.reward.item.ItemRequestDto;
+import com.agh.polymorphia_backend.dto.request.course.reward.item.ItemType;
+import com.agh.polymorphia_backend.dto.request.course.reward.item.PercentageBonusItemRequestDto;
+import com.agh.polymorphia_backend.dto.response.course.reward.item.FlatBonusItemResponseDto;
+import com.agh.polymorphia_backend.dto.response.course.reward.item.ItemResponseDto;
+import com.agh.polymorphia_backend.dto.response.course.reward.item.PercentageBonusItemResponseDto;
+import com.agh.polymorphia_backend.exception.database.InvalidArgumentException;
+import com.agh.polymorphia_backend.model.course.reward.Chest;
+import com.agh.polymorphia_backend.model.course.reward.item.FlatBonusItem;
+import com.agh.polymorphia_backend.model.course.reward.item.Item;
+import com.agh.polymorphia_backend.model.course.reward.item.PercentageBonusItem;
+import com.agh.polymorphia_backend.model.event.EventSection;
+import com.agh.polymorphia_backend.repository.event.EventSectionRepository;
+import lombok.AllArgsConstructor;
+import org.springframework.stereotype.Service;
+
+import java.util.HashSet;
+
+@Service
+@AllArgsConstructor
+public class ItemMapper {
+    private static final String BONUS_TEXT = "%s do eventów z kategorii %s";
+    private static final String XP_TEXT = "+%dxp";
+    private static final String PERCENT_TEXT = "+%d%%";
```

Comment:
```suggestion
    private static final String XP_TEXT = "+%d xp";
    private static final String PERCENT_TEXT = "+%d %%";
```
I think there should be a between value and unit.